tests/admin_views/models.py

```diff
@@ -330,6 +330,10 @@ class Meta:
         ordering = ('release_date',)  # overridden in PodcastAdmin
 
 
+class PodcastInfo(models.Model):
```

Comment:
I did look into using existing models, but decided to use one model that other `date_hierarchy` tests were using and create a related model.
I will take another look at it this weekend.